### PR TITLE
Add mainline installation to possibilites 

### PR DESCRIPTION
**Pull request by PumpkinSeed**
In this case you can install the latest nginx version from the mainline repository. From the original repository you are able to install 1.6 version while from the mainline it is possible to install the 1.9.x

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 - include: nginx-official-repo.yml
   when: nginx_official_repo == True
```
Comment (jdauphant) [inline tasks/main.yml]: nginx_official_repo == True and nginx_mainline_repo == False

File: tasks/main.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+- include: nginx-mainline-repo.yml
+  when: nginx_mainline_repo == True
```
Comment (jdauphant) [inline tasks/main.yml]: nginx_official_repo == True and nginx_mainline_repo == True

Comment (PumpkinSeed) [inline tasks/main.yml]: Why is it important? It is two separated, the mainline not depend from the use repo.

**Comment by jdauphant** [comment]
Hi @PumpkinSeed, thanks your contribution. Apparently there is some conflicts to resolve before merging.
I also made some comments to avoid nginx_official_repo and nginx_mainline_repo conflict.

**Comment by jdauphant** [comment]
@PumpkinSeed I just see that the same work in progress in a other PR : https://github.com/jdauphant/ansible-role-nginx/pull/107

**Comment by jdauphant** [comment]
@PumpkinSeed  I just merged the other PR : https://github.com/jdauphant/ansible-role-nginx/pull/107 .
Mainline is know supported in the last version.
Thanks for your help !

**Comment by PumpkinSeed** [comment]
I am thanks for it, because I would like to use it.

**Pull request closed without merging**